client/js/is-sum-formula.js

```diff
@@ -0,0 +1,52 @@
+const getColAndRow = function(cellId) {
+  const col = cellId[0];
+  const row = cellId.slice(1);
+  return [col, row];
+}
+
+const isValidSumFormula = function(string, numCols, numRows) {
+  // string must be in form
+  // =SUM([One capital letter][0 or more digits]:[One capital letter][0 or more digits])
+  const re = /=SUM\(([A-Z]\d*):([A-Z]\d*)\)$/;
+
+  // if string is in valid format, get col letters and row numbers
+  if (string.match(re) !== null) {
+    const startCellCol = getColAndRow(string.match(re)[1])[0].charCodeAt(0) - 64;
```

Comment:
Sorry to keep going back and forth on this, but you keep re-running the `match` for some reason.  You only need to call it once, and an object comes back that you can then pull the matched groups out of.  Please play around with this on the interactive command line (the Dev Tools console in the browser, or just type `node` on the command line) and you will see what I mean.  Hit us up on Slack if you don't get it.